airbyte-integrations/connectors/destination-mssql-v2/src/test-integration/resources/check/valid.json

```diff
@@ -6,5 +6,8 @@
   "raw_data_schema": "guest",
   "ssl_method": { "name": "unencrypted" },
   "user": "sa",
-  "password": "replace_me"
+  "password": "replace_me",
+  "load_type": {
+    "load_type": "INSERT"
```

Comment:
Can we add a test to make sure we can use the bulk load type, mostly making sure the config has been wired correctly. I have had issues with the generated code and had to use `lateinit` in some places for things to work.

Reply:
Oh we do have a integration test for this